tests/test_conan_version_range.py

```diff
@@ -0,0 +1,67 @@
+import pytest
```

Comment:
This file likely also comes from conan and likely therefore need a notice header and ABOUT file?

Reply:
All the tests that are re-used comes from https://github.com/conan-io/conan/tree/release/2.0/conans/test/unittests/model/version

Reply:
@pombredanne @TG1999 The .ABOUT files I've created so far for the non-test Conan files include a reference to corresponding .NOTICE files -- we want similar .NOTICE files for each of these three Conan test files too, right?

Reply:
I've answered for myself -- some other test files have both .ABOUT and .NOTICE files and I'll adopt that approach for Conan as well.